### PR TITLE
Add custom scope validator sample

### PR DESCRIPTION
**Pull request by sahandilshan**
## Purpose
* $subject

### CODE REVIEW

File: scope-validator/sample.scope.validator/src/main/java/org/wso2/sample/scope/validator/CustomScopeValidator.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    public boolean validateScope(OAuth2TokenValidationMessageContext oAuth2TokenValidationMessageContext) {
+
+        return true;
+    }
+
+    @Override
+    public String getName() {
+        return "CustomScopeValidator";
+    }
+
```
Comment (piraveena) [inline scope-validator/sample.scope.validator/src/main/java/org/wso2/sample/scope/validator/CustomScopeValidator.java:79]: remove new lines

File: scope-validator/sample.scope.validator/src/main/java/org/wso2/sample/scope/validator/CustomScopeValidator.java [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    @Override
+    public boolean validateScope(OAuth2TokenValidationMessageContext oAuth2TokenValidationMessageContext) {
+
+        return true;
+    }
+
+    @Override
+    public String getName() {
```
Comment (piraveena) [inline scope-validator/sample.scope.validator/src/main/java/org/wso2/sample/scope/validator/CustomScopeValidator.java:76]: Add new line after method signature

File: scope-validator/sample.scope.validator/src/main/java/org/wso2/sample/scope/validator/internal/CustomScopeValidatorDataHolder.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    private static CustomScopeValidatorDataHolder instance = new CustomScopeValidatorDataHolder();
+
+    private RealmService realmService;
+    private RegistryService registryService;
+
+    private CustomScopeValidatorDataHolder() {
+
+    }
+
+    public static CustomScopeValidatorDataHolder getInstance() {
```
Comment (piraveena) [inline scope-validator/sample.scope.validator/src/main/java/org/wso2/sample/scope/validator/internal/CustomScopeValidatorDataHolder.java:35]: Add new lines after method signatures. Fix in all places

File: scope-validator/sample.scope.validator/src/main/java/org/wso2/sample/scope/validator/internal/CustomScopeValidatorServiceComponent.java [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+    protected void setRealmService(RealmService rlmService) {
+
+        CustomScopeValidatorDataHolder.getInstance().setRealmService(rlmService);
+    }
+
+    protected void unsetRealmService(RealmService realmService) {
+
+        CustomScopeValidatorDataHolder.getInstance().setRealmService(null);
+    }
+
```
[the remark this answers is not in the document]
Comment (sahandilshan) [reply scope-validator/sample.scope.validator/src/main/java/org/wso2/sample/scope/validator/internal/CustomScopeValidatorServiceComponent.java]: Fixed with https://github.com/wso2/samples-is/pull/332/commits/3b4ef6b1c48bf7195e65db85fe9acdcb3c287c14

File: scope-validator/sample.scope.validator/src/main/java/org/wso2/sample/scope/validator/CustomScopeValidator.java (hunk 2 above)
Comment (sahandilshan) [reply scope-validator/sample.scope.validator/src/main/java/org/wso2/sample/scope/validator/CustomScopeValidator.java:76]: Fixed with https://github.com/wso2/samples-is/pull/332/commits/3b4ef6b1c48bf7195e65db85fe9acdcb3c287c14

File: scope-validator/sample.scope.validator/src/main/java/org/wso2/sample/scope/validator/CustomScopeValidator.java (hunk 1 above)
Comment (sahandilshan) [reply scope-validator/sample.scope.validator/src/main/java/org/wso2/sample/scope/validator/CustomScopeValidator.java:80]: Fixed with https://github.com/wso2/samples-is/pull/332/commits/3b4ef6b1c48bf7195e65db85fe9acdcb3c287c14

**Review by piraveena — APPROVED** [review]

**Pull request merged**